fw-child/resources/js/child-functions.js

```diff
@@ -530,4 +517,19 @@ const $ = jQuery;
 
     $('body').removeClass('spinner-on');
   });
+
+  window.addEventListener('load', function () {
```

Comment:
Is this essential? Was it not working before?

Please stick to jQuery for consistency. Use $(window).on('load', function () {})

Reply:
This piece of code was refactored and placed again on the vendor section. It no longer relies on the window.load event.